pkg/contracts/src/CVStrategy.sol

```diff
@@ -963,4 +874,95 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
         emit MinThresholdPointsUpdated(_minThresholdPoints, minThresholdPoints_);
         _minThresholdPoints = minThresholdPoints_;
     }
+
+    function disputeProposal(uint256 proposalId, string calldata context, bytes calldata _extraData)
```

Comment:
Considering this thread, maybe check for a dispute already exist to prevent redispute it when funding pool:
https://discord.com/channels/1195408490596933832/1268346556491239545

Reply:
as talked today on the call we need to implement a cooldown